src/sentry/scim/endpoints/users.py

```diff
@@ -0,0 +1,211 @@
+        except ResourceDoesNotExist:
+            return Response(SCIM_404_USER_RES, status=404)
+
+        context = serialize(member, serializer=OrganizationMemberSCIMSerializer())
+        return Response(context)
+
+    def patch(self, request, organization, member_id):
+        try:
+            member = self._get_member(organization, member_id)
+        except ResourceDoesNotExist:
+            return Response(SCIM_404_USER_RES, status=404)
+
+        for operation in request.data.get("Operations", []):
+            # we only support setting active to False which deletes the orgmember
+            if operation["value"]["active"] is False:
+                audit_data = member.get_audit_log_data()
+                if self._is_only_owner(member):
+                    return Response({"detail": ERR_ONLY_OWNER}, status=403)
+                with transaction.atomic():
+                    AuthIdentity.objects.filter(
+                        user=member.user, auth_provider__organization=organization
+                    ).delete()
+                    member.delete()
+                self.create_audit_entry(
+                    request=request,
+                    organization=organization,
+                    target_object=member.id,
+                    target_user=member.user,
+                    event=AuditLogEntryEvent.MEMBER_REMOVE,
+                    data=audit_data,
+                )
+                return Response(status=204)
+        context = serialize(member, serializer=OrganizationMemberSCIMSerializer())
+        return Response(context)
+
+    def delete(self, request, organization, member_id):
+        try:
+            member = self._get_member(organization, member_id)
+        except ResourceDoesNotExist:
+            return Response(SCIM_404_USER_RES, status=404)
+
+        audit_data = member.get_audit_log_data()
+        if self._is_only_owner(member):
+            return Response({"detail": ERR_ONLY_OWNER}, status=403)
+        with transaction.atomic():
+            AuthIdentity.objects.filter(
+                user=member.user, auth_provider__organization=organization
+            ).delete()
+            member.delete()
+        self.create_audit_entry(
+            request=request,
+            organization=organization,
+            target_object=member.id,
+            target_user=member.user,
+            event=AuditLogEntryEvent.MEMBER_REMOVE,
+            data=audit_data,
+        )
+
+        return Response(status=204)
+
+
+class OrganizationSCIMUserIndex(SCIMEndpoint):
+    def get(self, request, organization):
+        # note that SCIM doesn't care about changing results as they're queried
+        # TODO: sanitize get parameter inputs?
+
+        filter_val = parse_filter_conditions(request.GET.get("filter"))
+
+        queryset = (
+            OrganizationMember.objects.filter(
+                # Q(user__is_active=True) | Q(user__isnull=True),
```

Comment:
Is this supposed to be removed?

Reply:
good question, I need to fiddle with this query slightly. I _think_ we'll want inactive-at-user-level users to show up in scim results, and also filter out `InviteStatus = REQUESTED_TO_BE_INVITED and REQUESTED_TO_JOIN`